app/angular/src/server/angular-cli_config.js

```diff
@@ -110,7 +110,7 @@ export function applyAngularCliWebpackConfig(baseConfig, cliWebpackConfigOptions
   // because @angular-devkit/build-angular created rules have include/exclude for global style files.
   const rulesExcludingStyles = baseConfig.module.rules.filter(
     rule =>
-      !rule.test || (rule.test.toString() !== '/\\.css$/' && rule.test.toString() !== '/\\.scss$/')
+      !rule.test || (rule.test.toString() !== '/\\.css$/' && rule.test.toString() !== '/\\.sass$/' && rule.test.toString() !== '/\\.scss$/')
```

Comment:
I think it won't work now. Here is a string comparison, while we need a regex matching.

Reply:
Yes. please replace with something like `!rule.test.test('.sass')` etc